snapcraft/cli/_errors.py

```diff
@@ -14,12 +14,24 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import os
 import sys
 import traceback
 
 from . import echo
 from snapcraft.internal import errors
 
+import click
+# raven is not available on 16.04
+try:
+    from raven import Client as RavenClient
+    from raven.transport import RequestsHTTPTransport
+except ImportError:
+    RavenClient = None
+
+_MSG_INTERNAL_ERROR = 'Sorry, Snapcraft had an internal error.'
+_MSG_SEND_TO_SENTRY = 'To help us improve, would you like to send error data?'
```

Comment:
I would ask Mathew to improve the wording her.
It should mention that it's anonymous, that only the trace relevant to snapcraft and the snap being build will be collected, and that the info will never be shared outside of the snapcraft team. "error data" sounds scary to me.